packages/clay-badge/README.mdx

```diff
@@ -3,7 +3,7 @@ title: 'Badge'
 description: 'Badges help highlight important information, such as notifications or new and unread messages.'
 lexiconDefinition: 'https://liferay.design/lexicon/core-components/badges/'
 packageNpm: '@clayui/badge'
-sibling: 'docs/components/css-badges.html'
+siblings: ['docs/components/css-badges.html', 'docs/components/api-badges.html']
```

Comment:
are arrays permitted in frontmatter? I haven't seen it before, so I wasn't sure.